crate/crash/command.py

```diff
@@ -620,6 +624,23 @@ def save_and_exit():
 
 def _create_shell(crate_hosts, error_trace, output_writer, is_tty, args,
                   timeout=None, password=None):
+
+    # Explicit "timeout" function argument takes precedence.
```

Comment:
didn't dive into it, but where can `timeout` come from if not from the args?

Reply:
In this case, it can come from `args.timeout`, and/or from the dedicated keyword argument `timeout`.

Reply:
dedicated keyword timeout ? what do you mean, can you give an example from user's perspective?

Reply:
Hi. I can't provide a user example, but I can better outline the two possible ways of obtaining the timeout value in this case.

- One way is `args.timeout`, which carries the string value from the command line option `--timeout`.
- Further, I discovered a Python kwargs called `timeout=None`, which apparently can be used to directly obtain a Python timout value (scalar or object, no string).

![image](https://github.com/crate/crash/assets/453543/fe2f1699-eb42-4195-a7d2-e68f776165a5)

Reply:
It can always be used by the user via Python API. It does not necessarily needs to be used on the command line. Removing code will probably be a different patch though, as it will need to address backward-incompatibility concerns, at least as a changelog item.

Reply:
Ok, thx!